### DISCUSSION TITLE
`brew install curlpp` fails with "inreplace failed: expected replacement of ... with /usr/bin/clang "

### DISCUSSION THREAD

**Discussion by HFTrader** [Homebrew/discussions#1607] [Linux] [answered]
Is this something that might be on my machine only? 

![image](https://user-images.githubusercontent.com/11621271/120702499-c3ccd500-c479-11eb-95f9-d712ef1a40e3.png)

==> Downloading https://github.com/jpbarrette/curlpp/archive/v0.8.1.tar.gz
Already downloaded: /home/hbucher/.cache/Homebrew/downloads/a4cbcee7f6b21f4b5c1f70370098e487b1b9bfbd37205c6b17a17f9d2f54c9de--curlpp-0.8.1.tar.gz
==> cmake .
==> make install
Error: An exception occurred within a child process:
  Utils::Inreplace::Error: inreplace failed
/home/linuxbrew/.linuxbrew/Cellar/curlpp/0.8.1_1/bin/curlpp-config:
  expected replacement of "/home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/shims/mac/super/clang" with "/usr/bin/clang"

**Comment by gromgit** [comment, marked as answer]
It's not just you. Fix underway.

**Reply by gromgit** [reply]
@HFTrader Fixed in https://github.com/Homebrew/linuxbrew-core/pull/23419. `brew update` and try again.
